translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionCharSequenceAssertion.kt

```diff
@@ -19,6 +19,8 @@ enum class DescriptionCharSequenceAssertion(override val value: String) : String
     IGNORING_CASE("%s, Gross-/Kleinschreibung ignorierend"),
     MATCHES("stimmt vollständig überein mit"),
     MISMATCHES("stimmt nicht vollständig überein mit"),
+    NOT_FOUND("aber es wurde kein Treffer gefunden"),
+    NUMBER_OF_MATCHES_FOUND(/* TODO translate */ "and %s matches were found"),
```

Comment:
If this is satisfactory, then more translations are needed :smiley:

Reply:
`und %s Treffer wurden gefunden`